extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -81,6 +98,20 @@ class AmpAccordion extends AMP.BaseElement {
         content.setAttribute('aria-expanded', 'true');
       }
     }, content);
+    // TODO(zhouyx): ask if there's way to get index of currentTarget
+    // element directly without iterating all section.
+    var tempSession_ = '';
+    setPromise.then(() => {
+      this.sections_.forEach((section, index) => {
+        const contentIter = section.children[1];
+        if (contentIter.getAttribute('aria-expanded') == 'true') {
+          tempSession_ += '1';
+        } else {
+          tempSession_ += '0';
+        }
+      });
+      sessionStorage.setItem(this.id_, tempSession_);
```

Comment:
why do we need to do the write in the then()? why not do it inside of the mutateElement?